### PR TITLE
🔧 libs ui vue3

### PR DESCRIPTION
**Pull request by roiLeo**
ref #5611  (#6441 2nd round)

We might have a problem here, I can't manage to make oruga-next to work with our current stack.
I think we need to migrate first to Vue3 (main repo) to be able to use new NeoLib.

note: pwa module block app build ❓ 

#### Log Output
Capture d’écran 2023-08-07 à 8 05 27 PM

### CODE REVIEW

**Comment by yangwao** [comment]
vite 420


image

**Comment by yangwao** [comment]
since this is done, this can have progress 👀

- https://github.com/kodadot/nft-gallery/pull/6744
- #5852

**Comment by roiLeo** [comment]
> since this is done, this can have progress 👀

>  I think we need to migrate first to Vue3 (main repo) to be able to use new NeoLib. (oruga-next)

File: libs/ui/package.json [hunk 1]
```diff
@@ -10,18 +10,18 @@
...
-    "vue": "2.7.14"
+    "vite": "^4.2.0"
   },
   "dependencies": {
-    "@google/model-viewer": "^1.12.1",
     "bulma": "0.9.4",
-    "vue-audio-visual": "2.5.0"
+    "vue": "^3.2.47",
+    "vue-audio-visual": "2.5.0",
+    "@google/model-viewer": "^1.12.1"
```
Comment (vikiival) [inline libs/ui/package.json:25]: Why is this strangely sorted?

Comment (roiLeo) [reply libs/ui/package.json:25]: because adding dependencies manually hehe

File: components/TheFooter.vue [hunk 2]
```diff
@@ -2,10 +2,10 @@
   <footer class="footer-container section">
     <div
       class="footer-container-inner is-flex is-align-items-start is-justify-content-space-between">
-      <div class="footer-container-subs is-flex is-flex-direction-column">
-        <h4 class="subtitle is-5">
+      <section class="footer-container-subs is-flex is-flex-direction-column">
+        <h2 class="subtitle is-5">
```
Comment (vikiival) [inline components/TheFooter.vue:6]: why `h2` ? some more semantic css?

File: components/base/BaseCollectionForm.vue
Comment (vikiival) [inline components/base/BaseCollectionForm.vue:1]: maybe this as separated PR?

File: components/base/BaseTokenForm.vue
Comment (vikiival) [inline components/base/BaseTokenForm.vue:1]: LikeWise

File: components/TheFooter.vue (hunk 2 above)
Comment (yangwao) [reply components/TheFooter.vue:6]: mobile lighthouse maybb?

Comment (roiLeo) [reply components/TheFooter.vue:6]: wait what how do you see this changes?

**Comment by roiLeo** [comment]
This branch is for testing purposes imma try to big bang project to nuxt3

File: components/TheFooter.vue (hunk 2 above)
Comment (vikiival) [reply components/TheFooter.vue:6]: I have eyes. 
Equipped with best addon 
image

**Comment by preschian** [comment]
> This branch is for testing purposes imma try to big bang project to nuxt3

So excited this is coming 🤩
`"nuxt": "^3.6.3",`

> note: pwa module block app build ❓

Maybe just remove it if some package or some files is blocking the build? Remove what is blocked in this PR, and make a separate PR for the proper removal

**Comment by roiLeo** [comment]
Should I close this branch to continue in the other one?

**Comment by roiLeo** [comment]
continue in #7034

**Pull request closed without merging**